docs/csharp/language-reference/attributes/snippets/NullableAttributes.cs

```diff
@@ -0,0 +1,197 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace attributes
+{
+#nullable disable
+    public class NullableOblivious
+    {
+        // <TryGetExample>
```

Comment:
I had no idea this was an allowable way of doing code snippets. I'd always used regions.